packages/components/src/font-size-picker/font-size-picker-toggle-group.tsx

```diff
@@ -13,6 +13,23 @@ import {
 import { T_SHIRT_ABBREVIATIONS, T_SHIRT_NAMES } from './constants';
 import type { FontSizePickerToggleGroupProps } from './types';
 
+/**
+ * hasValidShortName
+ *
+ * check that the font size short name is a string with at most
+ * 3 characters length (e.g. "XXS", "XS", "S", "M", "L", "XL", "XXL")
+ *
+ * @param  shortName font size object
+ * @return boolean
+ */
+function hasValidShortName( shortName: string | undefined ): boolean {
+	return (
+		typeof shortName === 'string' &&
+		shortName.length >= 1 &&
+		shortName.length <= 3
+	);
+}
```

Comment:
Should we move this into the `utils.ts` file?

Reply:
Great call. I somehow missed that file. Done in [01ee1ec](https://github.com/WordPress/gutenberg/pull/46586/commits/01ee1ec588b9f580be9068651e8b73a178af314c)